src/applications/static-pages/widgetTypes.js

```diff
@@ -49,6 +49,7 @@ export default {
   FORM_107959C: 'form107959c',
   FORM_107959A: 'form107959a',
   FORM_107959F1: 'form107959F1',
+  FORM_107959F2: 'form107959F2',
```

Comment:
They want us to lower case the f for the second part here. I uppercased originally and was told to change it ASAP, but then they decided to leave it for now since I said I can, but it'll take a minute to get to staging, and they wanted it to be in staging immediately.